UDT.Model/Entities/Board.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UDT.Model.Enums;
+
+namespace UDT.Model.Entities
+{
+    public class Board
+    {
+        public int Id { get; set; }
+        public int TaskId { get; set; }
```

Comment:
Nu prea inteleg de ce avem si id-ul task-ului si task-ul in sine. Intrebarea se repeta si pentru user. 😃

Reply:
Ei bine ne trebuie amandoua, deoarece atunci cand iei entitatea din baza de date cu Entity Framework tu poti alege daca sa incluzi taskul sau nu, fiind o entitate separata de cea curenta. Poti alege sa nu o incluzi (in caz ca nu este nevoie) ca sa nu faci risipa de memorie si timp de executie, dar totusi sa te folosesti de id-ul ei, fiind deja in entitatea Board (sau User). Daca nu este inclus si id-ul pot aparea mici complicatii care fac sa fie necesare mai multe apeluri la baza de date, care de asemenea consuma mai mult timp. 

Si ai spus de partea cu creearea unei entitati ca sa nu fie nevoie sa adaugi toti parametrii in constructor. Well, in C# e o chestie faina, avand in vedere ca nu e un constructor explicit pentru entitate, tu alegi in mod direct ce parametrii dai in construcor si ce sa excluzi. De exemplu: 
```
Board board = new Board {
	TaskId = 1,
	UserId = 1,
	Status = TaskStatus.InProgress
}
```

sau poti de asemenea

```
Board board = new Board {
	TaskId = 1,
        Task = task
	UserId = 1,
	User = user,
	Status = TaskStatus.InProgress
}
```